run-jsondoclet.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Directory paths
+PROJECT_ROOT=$(pwd)
+TARGET_DIR="$PROJECT_ROOT/lucille-core/target"
+LIB_DIR="$TARGET_DIR/lib"
+CLASSES_DIR="$TARGET_DIR/classes"
+OUTPUT_DIR="$PROJECT_ROOT/lucille-plugins/lucille-api/src/main/resources"
```

Comment:
Looks like we're outputting directly into the source tree.
I think it would be cleaner to output into target.
If there's a need to output directly into source, we likely need to add these generated files to .gitignore